### PR TITLE
Fix and verify all broken links

### LINKED ISSUE #3044: Fix Broken Links
#### Description
This [spreadsheet](https://docs.google.com/spreadsheets/d/10t0NSlxQVbLS7_uPEJIY6AmkFib-HHqqj_UvGovJL8Q/edit?usp=sharing) contains a list of all broken links throughout the website.

#### Expected Behavior
Fix all broken links. Make sure to use the gatsby `<Link>` tag for all internal links. The spreadsheet contains the correct urls that can be used to replace these broken links. If a correct url is not present, simply remove the `<Link>` or `<a>` tags.

#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by JustinSalcedo**
**Description**

This PR fixes #3044, fixing **63** of the [69 broken links](https://docs.google.com/spreadsheets/d/10t0NSlxQVbLS7_uPEJIY6AmkFib-HHqqj_UvGovJL8Q/edit#gid=0) found across the website.

Many broken links were duplicated or affected by the same bug. I listed each broken link according to its line number.

2: 'Get the Book'
```diff
// src/collections/service-mesh-books/service-mesh-patterns/index.mdx
-  <Button primary className="banner-btn one pa-8" title="Get the Book" url="hhttps://www.oreilly.com/library/view/service-mesh-patterns/9781492086444/" external="true"/>
+  <Button primary className="banner-btn one pa-8" title="Get the Book" url="https://www.oreilly.com/library/view/service-mesh-patterns/9781492086444/" external="true"/>
```

[3, 14, 18, 31, 38]: '致力于解决服务网格使用中碰到的棘手问题'
```diff
// src/collections/events/kubecon-china-2021/index.mdx
- <h2><a href="https://sched.co/nPOK">致力于解决服务网格使用中碰到的棘手问题</a></h2>
+ <h2><a href="https://kccncosschn21.sched.com/event/pcbq/ji-jie-zhi-ke-zha-mu-solving-the-service-mesh-adopters-dilemma-anita-ihuman-layer5?iframe=no">致力于解决服务网格使用中碰到的棘手问题</a></h2>

// src/collections/events/kubecon-china-2021/index.mdx
- <h2><a href="https://sched.co/nPOK">致力于解决服务网格使用中碰到的棘手问题</a></h2>
+ <h2><a href="https://kccncosschn21.sched.com/event/pcbq/ji-jie-zhi-ke-zha-mu-solving-the-service-mesh-adopters-dilemma-anita-ihuman-layer5?iframe=no">致力于解决服务网格使用中碰到的棘手问题</a></h2>
```

[4, 15, 19, 32, 39]: 'CNCF TAG Network and Service Mesh Working Group'
```diff
// src/collections/events/kubecon-china-2021/index.mdx
- Talk Link: https://sched.co/pTl8
+ Talk Link: https://kccncosschn21.sched.com/event/pcYk/cncf-tag-jie-zhang-re-jie-yuan-cncf-tag-network-and-service-mesh-working-group-lee-calcote-layer5-ed-warnicke-cisco-ken-owens-fiserv?iframe=no

- <h2><a href="https://sched.co/pTl8">CNCF TAG Network and Service Mesh Working Group</a></h2>
+ <h2><a href="https://kccncosschn21.sched.com/event/pcYk/cncf-tag-jie-zhang-re-jie-yuan-cncf-tag-network-and-service-mesh-working-group-lee-calcote-layer5-ed-warnicke-cisco-ken-owens-fiserv?iframe=no">CNCF TAG Network and Service Mesh Working Group</a></h2>

// src/collections/events/kubecon-china-2021/index.mdx
- Talk Link: https://sched.co/pTl8
+ Talk Link: https://kccncosschn21.sched.com/event/pcYk/cncf-tag-jie-zhang-re-jie-yuan-cncf-tag-network-and-service-mesh-working-group-lee-calcote-layer5-ed-warnicke-cisco-ken-owens-fiserv?iframe=no

- <h2><a href="https://sched.co/pTl8">CNCF TAG Network and Service Mesh Working Group</a></h2>
+ <h2><a href="https://kccncosschn21.sched.com/event/pcYk/cncf-tag-jie-zhang-re-jie-yuan-cncf-tag-network-and-service-mesh-working-group-lee-calcote-layer5-ed-warnicke-cisco-ken-owens-fiserv?iframe=no">CNCF TAG Network and Service Mesh Working Group</a></h2>
```

5: "Contributor's Journey Map"
```diff
// src/sections/Community/Calendar/index.js
- Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="community/newcomers">Contributor's Journey Map </a>
+ Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="newcomers">Contributor's Journey Map </a>

// src/sections/Community/index.js
- Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="community/newcomers">Contributor's Journey Map </a>
+ Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="newcomers">Contributor's Journey Map </a>
```

[6, 7]: 'Link'
```diff
// src/collections/service-mesh-workshops/2019_10_15_all_things_open_raleigh/index.mdx
- eurl: https://allthingsopen.org/talk/using-service-meshes-workshop-extended-session/
+ eurl: https://2019.allthingsopen.org/talk/using-service-meshes-workshop-extended-session/

// src/collections/service-mesh-workshops/2020_01_16_cloud_native_austin/index.mdx
- eurl: https://allthingsopen.org/talk/using-service-meshes-workshop-extended-session/
+ eurl: https://2019.allthingsopen.org/talk/using-service-meshes-workshop-extended-session/
```

[8, 9]: Nighthawk Image
```diff
// src/sections/Community/Handbook/repo-data.js
- repository: "https://github.com/layer5io/nighthawk",
+ repository: "https://github.com/layer5io/getnighthawk",

- repository: "https://github.com/layer5io/smi-conformance",
+ repository: "https://github.com/layer5io/meshery-smi-conformance-action",
```

[10, 11, 12, 13, 16, 17, 25, 26, 27, 33, 36, 37]: 'Navendu Pottekkat'
```diff
// src/collections/blog/2021/2021-06-25-she-code-africa-contributhon-boot-camp/index.mdx
- <li><Link to="/community/members/navendu-pottekkat">Navendu Pottekkat</Link></li>
+ <li>Navendu Pottekkat</li>

// src/assets/data/tutorials/index.js
- link: "/community/members/navendu-pottekkat",

// src/collections/blog/2021/2021-06-25-she-code-africa-contributhon-boot-camp/index.mdx
- <li><Link to="/community/members/navendu-pottekkat">Navendu Pottekkat</Link></li>
+ <li>Navendu Pottekkat</li>

// src/collections/events/introduction-to-istio/index.mdx
- <Link to="/community/members/navendu-pottekkat">Navendu Pottekkat</Link>
+ Navendu Pottekkat

// src/collections/events/kubecon-china-2021/index.mdx
- <li><Link to="/community/members/navendu-pottekkat">Navendu Pottekkat</Link></li>
+ <li>Navendu Pottekkat</li>
```

[20, 21, 22]: 'Nighthawk Developers'; 'Nighthawk Maintainers'; 'Nighthawk Users'
```diff
- <li><a href="http://developers@nighthawk.dev">Nighthawk Developers</a></li>
- <li><a href="http://maintainers@nighthawk.dev">Nighthawk Maintainers</a></li>
- <li><a href="http://users@nighthawk.dev">Nighthawk Users</a></li>
+ <li><a href="https://groups.google.com/a/getnighthawk.dev/g/developers">Nighthawk Developers</a></li>
+ <li><a href="https://groups.google.com/u/2/a/getnighthawk.dev/g/maintainers">Nighthawk Maintainers</a></li>
+ <li><a href="https://groups.google.com/a/getnighthawk.dev/g/users">Nighthawk Users</a></li>
```

23: 'Nighthawk'
```diff
// src/collections/news/2022/2022-05-18-meshmark-the-cloud-native-value-measurement/meshmark-the-cloud-native-value-measurement.md
- With the project’s <Link to="/company/news/analyzing-service-mesh-performance">approach to performance analysis</Link> published in IEEE The Bridge, ongoing analysis of nearly 40,000 performance test results will comprise the initial release of the <a href="https://smp-spec.io/dashboard">Service Mesh Performance Dashboard</a>, also unveiling at KubeCon EU. <Link to="/cloud-native-management/meshery">Meshery</Link>, the cloud native manager, orchestrates the provisioning and configuration of Kubernetes and each service mesh. With <Link to="projects/nighthawk">Nighthawk</Link> embedded, Meshery’s performance management functions are what drives the tests run within the CNCF Lab.
+ With the project’s <Link to="/company/news/analyzing-service-mesh-performance">approach to performance analysis</Link> published in IEEE The Bridge, ongoing analysis of nearly 40,000 performance test results will comprise the initial release of the <a href="https://smp-spec.io/dashboard">Service Mesh Performance Dashboard</a>, also unveiling at KubeCon EU. <Link to="/cloud-native-management/meshery">Meshery</Link>, the cloud native manager, orchestrates the provisioning and configuration of Kubernetes and each service mesh. With <Link to="/projects/nighthawk">Nighthawk</Link> embedded, Meshery’s performance management functions are what drives the tests run within the CNCF Lab.
```

24: 'Layer5 technology partner'
```diff
// src/collections/news/2022/2022-05-10-layer5-joins-docker-extension-program-bringing-kubernetes-and-service-mesh-management-to-docker-with-meshery/index.mdx
- As a user of the Meshery Docker Extension, <Link to="/community/members/nic-jackson">Nic Jackson</Link>, Principal Developer Advocate of HashiCorp, can <Link to="/blog/docker/extending-the-docker-compose-experience-to-service-mesh">convert his Docker Compose apps to Meshery apps</Link>. Configuring and deploying Meshery apps to Kubernetes and any service mesh is made simple with the Meshery Docker Extension. HashiCorp is a <Link to="/news/layer5-and-hashicorp-launch-service-mesh-partnership">Layer5 technology partner</Link>.</p>
+ As a user of the Meshery Docker Extension, <Link to="/community/members/nic-jackson">Nic Jackson</Link>, Principal Developer Advocate of HashiCorp, can <Link to="/blog/docker/extending-the-docker-compose-experience-to-service-mesh">convert his Docker Compose apps to Meshery apps</Link>. Configuring and deploying Meshery apps to Kubernetes and any service mesh is made simple with the Meshery Docker Extension. HashiCorp is a <Link to="/company/news/layer5-and-hashicorp-launch-service-mesh-partnership">Layer5 technology partner</Link>.</p>
```

28: 'service mesh standards'
```diff
// src/sections/Community/Handbook/contributor-journey.js
- <Link to="../blog/announcements/a-standard-interface-for-service-meshes">
+ <Link to="/blog/announcements/a-standard-interface-for-service-meshes">
```

40: 'Service Mesh Interface standard'
```diff
// src/collections/blog/2021/2021-02-07-meshkit-and-meshery-adapter-library/index.mdx
- Meshery adapters are management plane components and manage the lifecycle of service meshes. This includes installation and deletion, configuration, and verification that an installation follows recommended practices. In addition, Meshery adapters can assess to what extent a service mesh complies to the <Link to="blog/announcements/a-standard-interface-for-service-meshes">Service Mesh Interface standard</Link>. Meshery adapters support management of multiple versions of their respective service mesh and also come bundled with sample applications that can be deployed for easy and quick exploration of service mesh capabilities. <div class="fact">Meshery adapters manage the lifecycle of service meshes.</div>
+ Meshery adapters are management plane components and manage the lifecycle of service meshes. This includes installation and deletion, configuration, and verification that an installation follows recommended practices. In addition, Meshery adapters can assess to what extent a service mesh complies to the <Link to="/blog/announcements/a-standard-interface-for-service-meshes">Service Mesh Interface standard</Link>. Meshery adapters support management of multiple versions of their respective service mesh and also come bundled with sample applications that can be deployed for easy and quick exploration of service mesh capabilities. <div class="fact">Meshery adapters manage the lifecycle of service meshes.</div>
```

41: 'partnerships'
```diff
// src/collections/blog/2020/2020-05-21-getting-started-with-mesheryctl/index.mdx
- Layer5 is a community-first, service mesh company which has technology <a href="partners">partnerships</a> with various tech giants like Microsoft, HashiCorp, CNCF, RedHat and many more to enlist. The community consists of open source leaders like maintainers of trending open-source projects, Google SoCers, Docker Captains, service mesh maintainers, Cloud Native Ambassadors and many more (<a href="http://slack.layer5.io">join in!</a>).
+ Layer5 is a community-first, service mesh company which has technology <a href="/partners">partnerships</a> with various tech giants like Microsoft, HashiCorp, CNCF, RedHat and many more to enlist. The community consists of open source leaders like maintainers of trending open-source projects, Google SoCers, Docker Captains, service mesh maintainers, Cloud Native Ambassadors and many more (<a href="http://slack.layer5.io">join in!</a>).
```

42: 'Maintainer Michael Gfeller'
```diff
// src/collections/blog/2021/2021-04-19-announcing-meshery-v0.5/index.mdx
- <div class="intro" style="text-align:center;margin: 2rem 4rem;"><p>Read <Link to="/community/member/michael-gfeller">Maintainer Michael Gfeller</Link>'s blog post <Link to="/blog/meshery/introducing-meshkit-and-the-meshery-adapter-library">Introduction MeshKit and the Meshery Adapter Library</Link> for more details.</p></div>
+ <div class="intro" style="text-align:center;margin: 2rem 4rem;"><p>Read <Link to="/community/members/michael-gfeller">Maintainer Michael Gfeller</Link>'s blog post <Link to="/blog/meshery/introducing-meshkit-and-the-meshery-adapter-library">Introduction MeshKit and the Meshery Adapter Library</Link> for more details.</p></div>
```

43: 'Implementing the Service Mesh Performance Specification'
```diff
// src/collections/blog/2020/2020-07-16-communitybridge-2020-teaching-service-meshes-to-be-compliant/index.mdx
- 3. **[Implementing the Service Mesh Performance Specification](/performance)** - Enhanced Meshery’s performance testing and profile management capabilities. Added support for customizing the requests made by the load generator, and added the capability of using performance test profiles in the tool.
+ 3. **[Implementing the Service Mesh Performance Specification](/projects/service-mesh-performance)** - Enhanced Meshery’s performance testing and profile management capabilities. Added support for customizing the requests made by the load generator, and added the capability of using performance test profiles in the tool.
```

[35, 44]: 'Meshery'; 'Service Mesh Performance Specification'
```diff
// src/collections/blog/2020/2020-06-23-google-summer-of-code-2020-service-mesh-performance-with-envoy-nighthawk/index.mdx
- My GSoC project is to <strong>Enable Distributed Load Testing for various Service Mesh Planes using Envoy Nighthawk</strong>. For those who are not familiar with <a href="https://github.com/envoyproxy/nighthawk">Nighthawk</a>, it is a performance benchmark tool written and maintained by the Envoy community. We are in the process of incorporating Nighthawk into <a href="/meshery">Meshery</a> - <i>the cloud native management plane</i>.Meshery already supports performance testing of service meshes using fortio & wrk2 and is maintained by the Layer5 community and is also a member of CNCF Landscape. Alongside this effort, we are creating a new industry standard: <a href="/projects/service-mesh-performance-specification">Service Mesh Performance Specification</a> which is another initiative of the Layer5 community to establish a common format for describing and capturing service mesh performance.
+ My GSoC project is to <strong>Enable Distributed Load Testing for various Service Mesh Planes using Envoy Nighthawk</strong>. For those who are not familiar with <a href="https://github.com/envoyproxy/nighthawk">Nighthawk</a>, it is a performance benchmark tool written and maintained by the Envoy community. We are in the process of incorporating Nighthawk into <a href="/meshery">Meshery</a> - <i>the cloud native management plane</i>.Meshery already supports performance testing of service meshes using fortio & wrk2 and is maintained by the Layer5 community and is also a member of CNCF Landscape. Alongside this effort, we are creating a new industry standard: <a href="/projects/service-mesh-performance">Service Mesh Performance Specification</a> which is another initiative of the Layer5 community to establish a common format for describing and capturing service mesh performance.
```

45: 'an article'
```diff
// src/collections/blog/2019/2019-10-02-hacktoberfest-2019-documenting-my-first-ever-hacktoberfest-contribution/index.mdx
- Before I continue on, I want to acknowledge the project that I have contributed to. In [an article](https://layer5.io/gsoc/2019) discussing their participating in the Google Summer of Code 2019 program, [**Layer5**](https://github.com/layer5io/layer5) is described as a community which represents "the largest collection of _service mesh_ projects and their maintainers in the world". [RedHat.com](https://www.redhat.com/en/topics/microservices/what-is-a-service-mesh) helpfully defines a service mesh as "a dedicated infrastructure layer" of an application that controls how different parts of that application ("services") share data with one another (i.e. 'mesh together').
+ Before I continue on, I want to acknowledge the project that I have contributed to. In [an article](/programs/gsod) discussing their participating in the Google Summer of Code 2019 program, [**Layer5**](https://github.com/layer5io/layer5) is described as a community which represents "the largest collection of _service mesh_ projects and their maintainers in the world". [RedHat.com](https://www.redhat.com/en/topics/microservices/what-is-a-service-mesh) helpfully defines a service mesh as "a dedicated infrastructure layer" of an application that controls how different parts of that application ("services") share data with one another (i.e. 'mesh together').
```

46: 'Lee Calcote'
```diff
// src/collections/blog/2022/2022-01-07-Recap-KubeCon-China-TAG-Network-talk/index.mdx
- The CNCF Technical Advisory Group for Network Co-Chairs <Link to="/members/lee-calcote">Lee Calcote</Link> and Ken Owens gave a presentation entitled Intro and Deep Dive into CNCF TAG Network and CNCF Service Mesh Working Group at KubeCon China 2021.
+ The CNCF Technical Advisory Group for Network Co-Chairs <Link to="/community/members/lee-calcote">Lee Calcote</Link> and Ken Owens gave a presentation entitled Intro and Deep Dive into CNCF TAG Network and CNCF Service Mesh Working Group at KubeCon China 2021.
```

[49, 51]: 'Previous chapter'; 'Getting Started'
```diff
// content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/deploy-an-application.mdx
- As part of Istio deployment in [Previous chapter](/learn/learning-paths/mastering-meshery-istio/introduction-to-service-meshes/istio/getting-started), you have deployed the sidecar injector.
+ As part of Istio deployment in [Previous chapter](../getting-started), you have deployed the sidecar injector.
```

52: 'Observability chapter'
```diff
// content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/circuit-breaking.mdx
- Before we can configure circuit breaking, please try to access the `product page` app from within `Meshery` to ensure all the calls are making it through **without** errors as we did in [Observability chapter](/learn/learning-paths/mastering-meshery-istio/introduction-to-service-meshes/istio/observability) (<a href={MesheryInitialLoadTest}>see screenshot</a>).
+ Before we can configure circuit breaking, please try to access the `product page` app from within `Meshery` to ensure all the calls are making it through **without** errors as we did in [Observability chapter](observability) (<a href={MesheryInitialLoadTest}>see screenshot</a>).
```

[53, 54]: 'Previous chapter'; 'Getting Started'
```diff
// content-learn/mastering-service-meshes-for-developers/advance-concepts-of-service-mesh/istio/deploy-an-application.mdx
- As part of Istio deployment in [Previous chapter](/learn/learning-paths/mastering-meshery-istio/advance-concepts-of-service-mesh/istio/getting-started), you have deployed the sidecar injector.
+ As part of Istio deployment in [Previous chapter](../getting-started), you have deployed the sidecar injector.

- Applying this yaml file included in the Istio package you collected in [Getting Started](/learn/learning-paths/mastering-meshery-istio/introduction-to-service-meshes/istio/getting-started#1) will deploy the BookInfo app in you cluster.
+ Applying this yaml file included in the Istio package you collected in [Getting Started](../getting-started) will deploy the BookInfo app in you cluster.
```

56: 'Previous chapter'
```diff
// content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/linkerd/deploy-an-application.mdx
- As part of Linkerd deployment in [Previous chapter](/learn/learning-paths/mastering-meshery-istio/introduction-to-service-meshes/linkerd/getting-started), you have deployed the sidecar injector.
+ As part of Linkerd deployment in [Previous chapter](../../istio/getting-started), you have deployed the sidecar injector.
```

57: 'localhost:9080'
```diff
// content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/linkerd/traffic-splitting.mdx
- Checking [localhost:9080](localhost:9080) would show you a product page, with a list of reviews on the right. Those reviews are being loaded from the reviews service which is backed by the 3 reviews pods. The requests to the reviews service are randomly sent to one of the 3 review pods, as they represent different versions of this service.
+ Checking localhost:9080 would show you a product page, with a list of reviews on the right. Those reviews are being loaded from the reviews service which is backed by the 3 reviews pods. The requests to the reviews service are randomly sent to one of the 3 review pods, as they represent different versions of this service.
```

58: 'here'
```diff
// src/collections/blog/2020/2020-05-21-getting-started-with-mesheryctl/index.mdx
- You can also provide a SMPS Configuration file with `perf` subcommand, with this file provided you will not have to specify url, duration, concurrent-requests & qps. However, if specified the value provided through file will be over-rided by value through CLI. For more info about file configuration, see [here](https://github.com/layer5io/service-mesh-performance-specification/blob/master/smps.yaml).
+ You can also provide a SMPS Configuration file with `perf` subcommand, with this file provided you will not have to specify url, duration, concurrent-requests & qps. However, if specified the value provided through file will be over-rided by value through CLI. For more info about file configuration, see [here](https://github.com/service-mesh-performance/service-mesh-performance/blob/master/docs/assets/spec/readme/service%20mesh%20performance%20specification%20result.yaml).
```

59: 'Meshery Documentation'
```diff
// src/collections/blog/2021/2021-04-19-announcing-meshery-v0.5/index.mdx
- To get a more comprehensive list of the bug fixes and enhancements packaged in the v0.5.0 release, see the [Meshery Documentation](https://docs.layer5.io/docs/project/releases). With v0.6.0 planning complete these signal of the Layer5 community's innovation cycle trending upward sharply. See Meshery's <Link to="https://github.com/layer5io/meshery/blob/master/ROADMAP.md">roadmap</Link> for a prelude of what's yet to come.
+ To get a more comprehensive list of the bug fixes and enhancements packaged in the v0.5.0 release, see the [Meshery Documentation](https://docs.meshery.io/). With v0.6.0 planning complete these signal of the Layer5 community's innovation cycle trending upward sharply. See Meshery's <Link to="https://github.com/layer5io/meshery/blob/master/ROADMAP.md">roadmap</Link> for a prelude of what's yet to come.
```

60: 'deliberate'; 'materialize.css'; 'incidental'
```diff
// src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx
- In [my last blog post](https://raungar.wordpress.com/2019/10/02/hacktoberfest-2019-documenting-my-first-ever-hacktoberfest-contribution/), I mentioned that [Materialize CSS](https://materializecss.com) was worth further research. Well, after some poking around, I discovered that all of Layer5.io's pumpkin-colored hyperlink styling (both [deliberate](https://github.com/layer5io/layer5/blob/master/assets/css/materialize.css#L1751), and [incidental](https://github.com/layer5io/layer5/blob/master/assets/css/materialize.css#L5149)) originated from its [materialize.css](https://github.com/layer5io/layer5/blob/master/assets/css/materialize.css) asset.
+ In [my last blog post](https://raungar.wordpress.com/2019/10/02/hacktoberfest-2019-documenting-my-first-ever-hacktoberfest-contribution/), I mentioned that [Materialize CSS](https://materializecss.com) was worth further research. Well, after some poking around, I discovered that all of Layer5.io's pumpkin-colored hyperlink styling (both deliberate, and incidental) originated from its materialize.css asset.
```

[61, 62]: 'Istio Website'; ' Twitter'
```diff
// src/collections/blog/2020/2020-07-18-service-mesh-patterns-for-multitenancy/index.mdx
- Reference: [Istio Website](https://istio.io/latest/docs/setup/install/standalone-operator/)
+ Reference: [Istio Website](https://istio.io/latest/docs/setup/install/)

- <a href="https://twitter/sudeepbatra"> Twitter</a>.
+ <a href="https://twitter.com/sudeepbatra"> Twitter</a>.
```

[63, 64]: 'service mesh landscape'; 'getting started with Meshery'
```diff
// src/collections/blog/2020/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx
- Released on August 5th, 2020 by Microsoft, [Open Service Mesh](https://openservicemesh.io/) (OSM) is a lightweight and [Service Mesh Interace conformant](https://layer5.io/smi) (SMI). Open Service Mesh is a contemporary addition to the [service mesh landscape](https://github.com/layer5io/layer5/blob/master/landscape). Using Envoy as its data plane proxy component and SMI specifications as it's control plane APIs, OSM draws lessons and code from existing service mesh projects, like Linkerd. The Open Service Mesh project has some miles to go as it is one of a growing list of choices available in the service mesh landscape.
+ Released on August 5th, 2020 by Microsoft, [Open Service Mesh](https://openservicemesh.io/) (OSM) is a lightweight and [Service Mesh Interace conformant](https://layer5.io/smi) (SMI). Open Service Mesh is a contemporary addition to the [service mesh landscape](/service-mesh-landscape). Using Envoy as its data plane proxy component and SMI specifications as it's control plane APIs, OSM draws lessons and code from existing service mesh projects, like Linkerd. The Open Service Mesh project has some miles to go as it is one of a growing list of choices available in the service mesh landscape.

- Try Open Service Mesh now, by [getting started with Meshery](https://github.com/layer5io/layer5/blob/master/meshery#getting-started).
+ Try Open Service Mesh now, by [getting started with Meshery](/cloud-native-management/meshery/getting-started).
```

[65, 66]: 'Settings page'; 'Performance page'
```diff
// content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/observability.mdx
- On the [Settings page](http://localhost:9081/settings#metrics):
+ On the Settings page:

- On the [Performance page](http://localhost:9081/performance):
+ On the Performance page:
```

[67, 68]: 'Settings page'; 'Performance page'
```diff
// content-learn/mastering-service-meshes-for-developers/advance-concepts-of-service-mesh/istio/observability.mdx
- On the [Settings page](http://localhost:9081/settings#metrics):
+ On the Settings page:

- On the [Performance page](http://localhost:9081/performance):
+ On the Performance page:
```

70: 'http://kubernetes.docker.internal/'
> Gatsby automatically renders valid URL strings, so I had to split this unintended link with a `<span>`
```diff
// content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/linkerd/expose-services.mdx
- You may use http://localhost or http://kubernetes.docker.internal or your host's IP address.
+ You may use http://localhost, <span>http://</span>kubernetes.docker.internal or your host's IP address.
```

**Notes for Reviewers**
Links in lines 34, 47, 48, and 50 were already replaced with the correct URLs, probably by the same members affected.
The spreadsheet does not specify URL replacements for links 55 and 69, which refer to https://www.alldaydevops.com/addo-speakers/lee-calcote (deleted)

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Community/Calendar/index.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ const Calendar = () => {
               <h1>Newcomers Welcome!</h1>
               <h3>Are you new to the community?</h3>
               <p className="invitation">
-                Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="community/newcomers">Contributor's Journey Map </a>
+                Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="newcomers">Contributor's Journey Map </a>
```
Comment (debo19) [inline src/sections/Community/Calendar/index.js:146]:
```suggestion
                Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <Link to="/community/newcomers">Contributor's Journey Map </Link>
```

File: src/sections/Community/index.js [hunk 2]
```diff
@@ -138,7 +138,7 @@ const CommunityPage = ({ theme }) => {
                 <h1>Newcomers Welcome!</h1>
                 <h3>Are you new to the community?</h3>
                 <p className="invitation">
-                  Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="community/newcomers">Contributor's Journey Map </a>
+                  Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <a href="newcomers">Contributor's Journey Map </a>
```
Comment (debo19) [inline src/sections/Community/index.js:141]:
```suggestion
                  Begin your journey by <a href="http://slack.layer5.io">joining the community Slack</a>. Then, use the resources linked in our <Link to="/community/newcomers">Contributor's Journey Map </Link>
```

Comment (debo19) [reply src/sections/Community/index.js:141]: Use `<Link>` tag for all internal links

**Review by debo19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 2fc66ad264ea7214e4591babe165a85396c25c57 at: https://6304bce98e94907034f04904--layer5.netlify.app

File: src/collections/events/introduction-to-istio/index.mdx [hunk 3]
```diff
@@ -46,7 +46,7 @@ Lee Calcote walks you through advanced service mesh concepts and each and every
 
 - Docker (Docker Desktop for Mac or Windows, or Docker for Linux)
 - Kubernetes (using Docker Desktop or Minikube for Mac, Windows, or Linux)
-- [Meshery](/projects/meshery) (Mac, Windows or Linux)
+- [Meshery](/service-mesh-management/meshery) (Mac, Windows or Linux)
```
Comment (debo19) [inline src/collections/events/introduction-to-istio/index.mdx:49]:
```suggestion
- [Meshery](/cloud-native-management/meshery) (Mac, Windows or Linux)
```

**Review by debo19 — CHANGES_REQUESTED** [review]

File: content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/linkerd/expose-services.mdx [hunk 4]
```diff
@@ -86,7 +86,7 @@ To test this, you need to get the external IP of your controller.
 
 <h4 class="chapter-sub-heading"> Docker Desktop</h4>
 
-You may use http://localhost or http://kubernetes.docker.internal or your host's IP address.
+You may use http://localhost, <span>http://</span>kubernetes.docker.internal or your host's IP address.
```
[the remark this answers is not in the document]
Comment (JustinSalcedo) [reply content-learn/mastering-service-meshes-for-developers/introduction-to-service-meshes/linkerd/expose-services.mdx:89]: This is an English grammar correction, but I can change it back anyways.

**Comment by l5io** [comment]
🚀 Preview for commit d97693f8fb37be744e97bf5e512dc14caec16993 at: https://63052c582f395500b693e0c4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 706a1ade0c6eed6a588bdd5df1483286c3597a4d at: https://6307531fff7afd0304235580--layer5.netlify.app

**Review by debo19 — APPROVED** [review]
LGTM 🎉 Thank you @JustinSalcedo

**Comment by JustinSalcedo** [comment]
You're all welcome, @debo19 . What a pleasure!

**Pull request merged**